plugins/task/updatenotification/services/provider.php

```diff
@@ -27,16 +27,18 @@
      *
      * @return  void
      *
-     * @since   4.4.0
+     * @since   __DEPLOY_VERSION__
      */
-    public function register(Container $container): void
+    public function register(Container $container)
```

Comment:
add return type again
```suggestion
    public function register(Container $container): void
```